into/backends/spark.py

```diff
@@ -32,6 +34,11 @@ def anything_to_spark_context(sc, o, **kwargs):
     return append(sc, convert(list, o, **kwargs), **kwargs)
 
 
+@append.register(RDD, RDD)
+def append_rdd_to_rdd(a, b, **kwargs):
+    return a.union(b)
```

Comment:
We should be aware that this breaks conventions of `append` (which are pretty loose anyway) as an operation that mutates the left hand side.